Dockerfile

```diff
@@ -0,0 +1,11 @@
+# Create image based on the official Node image from dockerhub
+FROM node:18
```

Comment:
Should this be 16? That's what the CLI is on currently

Reply:
Good call, Fixed in my latest commit